humans.txt

```diff
@@ -0,0 +1,52 @@
+/* humanstxt.org */
+
+
+/* TEAM */
+	Advisor: Foster McLane
+	Contact: fmclane@clemson.edu
+
+	President: Matthew Rinaldi
+	Contact: mjrinal@g.clemson.edu
+
+	Vice President: Matt Ehrnschwender
+	Contact: mehrnsc@g.clemson.edu
+
+	Treasurer: Madeleine Wolfe
+	Contact: mgwolfe@g.clemson.edu
+
+	Secretary: Kaitlyn Pierce
+	Contact: kjpierc@g.clemson.edu
+
+	Training Organizer: Matthew Burnett
+	Contact: mburne3@g.clemson.edu
+
+	Competition and Events Organizer: David Houston
+	Contact: dghoust@g.clemson.edu
+
+
+/* THANKS */
+	Weston Belk
+	Mackenzie Binns
+	Nick Bulischeck
+	Sean Craffey
+	Max Harley
+	Morgan McEntire
+	Cameron Ogle
+	Thomas Rea
+	Derek Rodriguez
+	Andrew Samuels
+	Kenneth Simpson
+	Robert Underwood
+	Adam Van Scyoc
+	Kyle Wolfe
+	Noah Wright
+
+
+/* Disclosures */
```

Comment:
Either this should be all caps (which seems to be a convention) or the other headings need to be title-cased. I've no preference.

Reply:
Changed to be ALL UPPERCASE in 58f0542